.github/workflows/deploy.yml

```diff
@@ -38,5 +38,8 @@ jobs:
             --name "Stable Diffusion" \
             --env RATE_LIMIT_KEY=${{secrets.RATE_LIMIT_KEY}} \
             --env SHEET_API_URL=${{secrets.SHEET_API_URL}} \
-            --env MUSE_ROOT_PATH="/muse"
+            --env MUSE_ROOT_PATH="/muse" \
+            --env MUSE_MIN_WORKERS=5 \
```

Comment:
can we set this as input with default, see: https://dev.to/mrmike/github-action-handling-input-default-value-5f2g

Reply:
@Borda resolved!